apps/webapp/app/send/ibc-form.tsx

```diff
@@ -0,0 +1,116 @@
+import dynamic from 'next/dynamic';
+import { LoHi, uint8ArrayToBase64 } from 'penumbra-types';
+import { useEffect } from 'react';
+import { Button, Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from 'ui';
+import { useBalances } from '../../hooks/balances';
+import { FilledImage } from '../../shared';
+import { useStore } from '../../state';
+import { ibcSelector } from '../../state/ibc';
+import { calculateBalance, validateAmount } from '../../utils';
+import { chains } from './constants';
+const InputToken = dynamic(() => import('../../shared/input-token'), {
+  ssr: false,
+});
+
+export default function IbcForm() {
```

Comment:
https://github.com/penumbra-zone/web/assets/16624263/a457a37b-920a-4cf1-b3e8-c0f950988cfa

There appears to be a flicker for this component.

Reply:
This is due to dynamic imports! looking for a solution

Reply:
Ah ok. It may be trickier to fix than I thought. Feel free to skip this if it seems too difficult for now.